Source/Flow/Private/Nodes/FlowNode.cpp

```diff
@@ -112,7 +112,7 @@ void UFlowNode::AddOutputPins(TArray<FFlowPin> Pins)
 
 bool UFlowNode::RebuildPinArray(const TArray<FName>& NewPinNames, TArray<FFlowPin>& InOutPins, const FFlowPin& DefaultPin)
 {
-	bool bIsChanged = false;
```

Comment:
Oh.  I didn't mean to change this from mainline.  I thought mainline removed the false.  ('Resharper' will sometimes remove default values like this if all 'if' branches set the value)

I'll put it back in.

Reply:
Yes, I removed the false as static analysis pointed that out.